android/app/google-services.json

```diff
@@ -0,0 +1,48 @@
+{
+  "project_info": {
+    "project_number": "276400703284",
+    "firebase_url": "https://test-947g5.firebaseio.com",
+    "project_id": "test-947g5",
+    "storage_bucket": "test-947g5.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:276400703284:android:38a365ea5c098b16330761",
+        "android_client_info": {
+          "package_name": "com.myawesomeapp"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "276400703284-s7gori09m2bro8h5t2b8djeo93r5piso.apps.googleusercontent.com",
+          "client_type": 1,
+          "android_info": {
+            "package_name": "com.myawesomeapp",
+            "certificate_hash": "def5c563fff2df25639a41c725e5958af8d93e10"
+          }
+        },
+        {
+          "client_id": "276400703284-tt46qv5k68bl82u1olocq7435oor38dp.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyBv75JwZ1hMx7Jo_DYmKmNTQLGv1uIl6_g"
```

Comment:
Shouldn't this be secret?

Reply:
How do you intend I go about it?
It has to be in this file or we can encrypt somehow